data/json/furniture.json

```diff
@@ -1949,6 +2012,7 @@
         "type" : "furniture",
         "id" : "f_forge",
         "name": "forge",
+        "description": "There's my lava!",
```

Comment:
Funny, but won't mean anything to players without an anvil nearby. Recommend you add some description of what can be done with it.